distribution/tools/plugin-cli/src/main/java/org/opensearch/plugins/InstallPluginCommand.java

```diff
@@ -121,21 +121,21 @@
  * The installation process first extracts the plugin files into a temporary
  * directory in order to verify the plugin satisfies the following requirements:
  * <ul>
- * <li>Jar hell does not exist, either between the plugin's own jars, or with opensearch</li>
- * <li>The plugin is not a module already provided with opensearch</li>
+ * <li>Jar hell does not exist, either between the plugin's own jars, or with OpenSearch</li>
+ * <li>The plugin is not a module already provided with OpenSearch</li>
  * <li>If the plugin contains extra security permissions, the policy file is validated</li>
  * </ul>
  * <p>
  * A plugin may also contain an optional {@code bin} directory which contains scripts. The
- * scripts will be installed into a subdirectory of the opensearch bin directory, using
+ * scripts will be installed into a subdirectory of the OpenSearch bin directory, using
  * the name of the plugin, and the scripts will be marked executable.
  * <p>
  * A plugin may also contain an optional {@code config} directory which contains configuration
  * files specific to the plugin. The config files be installed into a subdirectory of the
- * opensearch config directory, using the name of the plugin. If any files to be installed
+ * OpenSearch config directory, using the name of the plugin. If any files to be installed
  * already exist, they will be skipped.
  */
-class InstallPluginCommand extends EnvironmentAwareCommand {
+public class InstallPluginCommand extends EnvironmentAwareCommand {
```

Comment:
Why we making those public?

Reply:
The price of opening things up for javadocs to  be published is not justifiable (to me). We don't want to publish (and promote) the things which are not supposed to be public.

Reply:
Got it, there's 2 other classes in the same package that are publicly visible: [PluginHelper](https://github.com/opensearch-project/OpenSearch/blob/main/distribution/tools/plugin-cli/src/main/java/org/opensearch/plugins/PluginHelper.java) and [PluginCli](https://github.com/opensearch-project/OpenSearch/blob/main/distribution/tools/plugin-cli/src/main/java/org/opensearch/plugins/PluginCli.java). Do you think the visibility of those are fine or should those also be package-private?

Reply:
Thanks @cwperks , PluginCli seems to be runnable (`main`) so it sadly needs to be public, but PluginHelper's visibility could be reduced I think.